src/Locale/de/cake.po

```diff
@@ -246,7 +246,7 @@ msgstr "Start"
 
 #: View/Widget/DateTimeWidget.php:540
 msgid "January"
-msgstr "Januar"
+msgstr "Jänner"
```

Comment:
@mdlouhy thank your for your work.

I believe you that in Austria you *say* "Jänner", because sone people in Switzerland, do too  (depending on the dialect).  😼
But I doubt that this would be normal for *formal* *written* Austrian German, as for example in a news paper.

And since a ``de_AT`` locale in a web application would imply *written* and most often *formal* Austrian German, I think it would still be "Januar".